src/app/artist-details/artist-details.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="card2" *ngIf="artist">
```

Comment:
Please move all styling logic into CSS files unless styling needs to be applied dynamically,

Reply:
@strizzwald fixed